sdk/storage/azblob/CHANGELOG.md

```diff
@@ -1,14 +1,17 @@
 # Release History
 
-## 0.5.0 (Unreleased)
+## 0.5.0 (Beta)
 
 ### Breaking Changes
 
 * Complete architectural change for better user experience. Please see the example file for more information.
+* Added [UserDelegationCredential](https://learn.microsoft.com/en-us/rest/api/storageservices/create-user-delegation-sas) which resolves [#18976](https://github.com/Azure/azure-sdk-for-go/issues/18976), [#16916](https://github.com/Azure/azure-sdk-for-go/issues/16916), [#18977](https://github.com/Azure/azure-sdk-for-go/issues/18977) 
+* Added [Restore Container API](https://learn.microsoft.com/en-us/rest/api/storageservices/restore-container).
 
 ### Bugs Fixed
 
 * Fixed bugs reported in the previous version of `azblob`.
```

Comment:
This entry is a bit vague, maybe we can remove it.

There was also fix https://github.com/Azure/azure-sdk-for-go/pull/16937 that I merged a few days ago.  There's no issue, but probably worth mentioning.